openfusion_ros:
  openfusion_ros:
    ros__parameters:
      robot:
        pose_source: "topic"  # options: "tf" or "topic"
        # If pose_source is "topic", set the following topic:
        pose_topic: "/openfusion_ros/camera_pose"
        # If pose_source is "tf", set the following frames:
        parent_frame: "map"
        child_frame: "carter_camera_stereo_right"
        max_delta_time: 0.2
        camera:
          rgb_topic: "/rgb"
          depth_topic: "/depth"
          debug_images: false
          max_buffer_size: 50
          max_buffer_age_sec: 3.0
      append_poses_frequency: 10.0
      pointcloud_frequency: 1.0
      pose_min_translation: 0.05
      pose_min_rotation: 5.0
      topk: 20
      depth_max: 10.0
      use_sim_time: true
      skip_loading_model: false
      min_inferno_score: 0.2
      max_inferno_score: 0.45

openfusion_ros:
  openfusion_tf_bridge:
    ros__parameters:
      robot:
        map_frame: "map"
        camera_frame: "carter_camera_stereo_right"
      camera_pose_topic: "camera_pose"
      rate_hz: 30.0